tests/spec/puppeteer-axe-spec.js

```diff
@@ -0,0 +1,37 @@
+/* Puppeteer tests for the A11y. */
+const { AxePuppeteer } = require('axe-puppeteer');
+const puppeteer = require('puppeteer');
+
+describe("Puppeteer accessibility tests for the Python Editor.", function() {
+    'use strict';
+
+    const editorURL = 'http://localhost:5000/editor.html';
+    let browser = null;
+
+    beforeAll(async () => {
+        // Setup a headless Chromium browser.
+        // Flags allow Puppeteer to run within a container.
+        browser = await puppeteer.launch({
+            headless: true,
+            args: ["--no-sandbox", "--disable-setuid-sandbox", "--disable-dev-shm-usage"]
+        });
+        const page = await browser.newPage();
+ 
+        await page.goto(editorURL);    
+ 
+        await page.close();
+        });
+
+    afterAll(async () => {
+    browser.close();
+    });
+
+    test( 'Checks the editor.html page with Axe', async () => {
+        // First, run some code which loads the content of the page.
+        const page = await browser.newPage();
+        await new AxePuppeteer(page).analyze();
+        await expect(page).toPassAxeTests({
+            disabledRules: [ 'document-title', 'page-has-heading-one', 'html-has-lang', 'landmark-one-main' ],
```

Comment:
`html-has-lang`:
> Ensures every HTML document has a lang attribute

We have a default lan in the HTML element, shouldn't this pass as well?

Reply:
Not sure on this one. `html-lang-valid` does pass which would indicate that the test is picking up the lang attribute. Running it against python.microbit.org does not pass either.

Reply:
There is no lang attribute in the versioner, since we cannot really change it from the editor.
But if these tests are for the editor we should remove this rule.

Reply:
As comment above, was not checking against editorURL